var-test-support/src/main/java/io/varhttp/test/VarClient.java

```diff
@@ -11,4 +15,34 @@ public interface VarClient {
 	VarClientRequest get(String path);
 	VarClientRequest delete(String path);
 	VarClientRequest head(String path);
+
+	static HttpResponse getHttpResponseWithContent(VarClientRequest.ContentFormat contentFormat, HttpURLConnection conn) throws IOException {
```

Comment:
considering this a static helper, can we consider moving it somewhere else to not polute the interface?

Reply:
maybe the HttpClient can take care of it completely